admin-action/package.json.liquid

```diff
@@ -6,21 +6,21 @@
 {%- if flavor contains "preact" %}
   "dependencies": {
     "preact": "^10.10.x",
-    "@shopify/ui-extensions": "~2025.10.0-rc"
```

Comment:
This one should stay the same. And ditto for all of the `if flavor contains "preact"` blocks